cypress/data/index.js

```diff
@@ -1,18 +1,32 @@
 import {
+    DIMENSION_ID_EVENT_STATUS,
+    DIMENSION_ID_PROGRAM_STATUS,
+    DIMENSION_ID_CREATED_BY,
+    DIMENSION_ID_LAST_UPDATED_BY,
     DIMENSION_ID_EVENT_DATE,
     DIMENSION_ID_ENROLLMENT_DATE,
     DIMENSION_ID_INCIDENT_DATE,
+    DIMENSION_ID_SCHEDULED_DATE,
     DIMENSION_ID_LAST_UPDATED,
 } from '../../src/modules/dimensionConstants.js'
 import { getPreviousYearStr } from '../helpers/period.js'
 
+const mainDimensions = {
```

Comment:
I don't see the need to append the main dimension names to the different programs, as the names are static and won't change (compared to e.g. `eventDate` that is dynamic).
On further thought, I don't see the need to keep the `mainDimensions` here in `/data/index.js` either, as its only used in the one place (so far). No need to over-complicate things.

Reply:
Refactored.